dotnetv3/ACM/README.md

```diff
@@ -0,0 +1,41 @@
+# ACM code examples for the AWS SDK for .NET v3
+
+## Overview
+
+This folder contains examples for using the AWS Certificate Manager (ACM) using the AWS SDK for .NET v3.
+
+The ACM creates private certificates for your internal resources and manage the certificate lifecycle centrally.
```

Comment:
This should typically be the blurb from the landing page for the service, not something we write ourselves. In this case, from [ACM landing page](https://docs.aws.amazon.com/acm/?id=docs_gateway):

> AWS Certificate Manager (ACM) makes it easy to provision, manage, and deploy SSL/TLS certificates on AWS managed resources.